ROOTFS/usr/bin/waggle-network-watchdog

```diff
@@ -123,67 +154,144 @@ def reboot_os():
     # aggressively but safely reboot the system
     subprocess.run(["systemctl", "--force", "reboot"])
 
+def shutdown_os():
```

Comment:
make sure nx watchdog doesn't reset it (Sean)

Reply:
Thanks! I'm not sure what the upper bound on that watchdog is, but 15 minutes seems promising.